File: Baekjoon/Class1/28.10250 ACM 호텔.py
import sys
input = sys.stdin.readline

t = int(input())
for _ in range(0, t):
    h, w, n = map(int, input().split())
    y = (n - 1) % h + 1
    x = (n - 1) // h + 1
    print(y * 100 + x)

File: Baekjoon/Class1/28.10250 ACM 호텔.js
const inputFile = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = require("fs").readFileSync(inputFile, "utf8").trim().split(/\s+/).map(Number);

let cursor = 0;
const t = input[cursor++];
const out = [];
for(let _ = 0; _ < t; _++) {
    const h = input[cursor++];
    const w = input[cursor++];
    const n = input[cursor++];
    const y = (n - 1) % h + 1;
    const x = Math.floor((n - 1) / h + 1);
    out.push(y * 100 + x);
}
console.log(out.join("\n"));